### DISCUSSION TITLE
/command/set_temps?driver_temp=22.0&passenger_temp=22.0 always set temperature to low（15℃）

### DISCUSSION THREAD

**Discussion by LQCross** [timdorr/tesla-api#611] [Q&A]
/command/set_temps?driver_temp=22.0&passenger_temp=22.0 always set temperature to low（15℃）
Please help me.
Thanks a lot.

**Comment by core-hacked** [comment]
@LQCross try to send the parameters with json inside the post body
```json
{
    "driver_temp": "22.0",
    "passenger_temp": "22.0"
}
```
